chsdi/templates/htmlpopup/windatlas50.mako

```diff
@@ -230,92 +232,92 @@ td.inner-table {
         </tr>
         <tr>
           <td>total</td>
-          <td>${round(freq_total, 3)} %</td>
-          <td>${round(props['v_mean'], 3)}</td>
-          <td>${round(props['wei_a'], 3)}</td>
-          <td>${round(props['wei_k'], 3)}</td>
+          <td id="align-right">${round(freq_total, 1)} %</td>
```

Comment:
Or you can use parent class `td.kernkraftwerke-extended > .myrighalign`. I'm sure there's a way.